pom.xml

```diff
@@ -631,6 +631,13 @@
                                 </execution>
                             </executions>
                         </plugin>
+                        <plugin>
+                            <groupId>org.apache.felix</groupId>
+                            <artifactId>maven-bundle-plugin</artifactId>
+                            <configuration>
+                                <obrRepository>NONE</obrRepository>
```

Comment:
Can't we add this to the bundle plugin config that already exists further up around line 300?